src/main.cpp

```diff
@@ -4954,9 +4960,13 @@ bool ContextualCheckBlockHeader(const CBlockHeader& block, CValidationState& sta
                         "rejected no static reward block");
 
    if((block.nVersion & nCFundReducedQuorumMask) != nCFundReducedQuorumMask && IsReducedCFundQuorumEnabled(pindexPrev,Params().GetConsensus()))
-     return state.Invalid(false, REJECT_OBSOLETE, strprintf("bad-version(0x%08x)", block.nVersion),
+       return state.Invalid(false, REJECT_OBSOLETE, strprintf("bad-version(0x%08x)", block.nVersion),
                         "rejected no reduced quorum block");
 
+   if((block.nVersion & nV451ForkMask) != nV451ForkMask && pindexPrev->nHeight >= Params().GetConsensus().nHeightv451Fork)
+       return state.Invalid(false, REJECT_OBSOLETE, strprintf("bad-version(0x%08x)", block.nVersion),
+                        "rejected no v4.5.1 block");
```

Comment:
```suggestion
                        "rejected, block version isn't v4.5.1");
```

Reply:
This seems to be the rejection message for when we get a block, if so, this wording might be better